ONSdigital/sdc-responses-dashboard
# Initial setup of Node & Babel

### Motivation and Context
Have made use of Babel to enable higher backwards compatibility.
Going forward, we can look at managing the existing dependencies via Node as well.

### What has changed
Added config for babel and minification.
For now, the js files will be transpiled into ES5 individually.
This is the case for minification of JS/CSS also.

Once the CSS/JS is minified it is exported into the `static/dist/` folder. References to assets will be made using this.
The dependencies and static minified assets are created on the fly and not stored in the repo.
 
### How to test?
- Ensure that the app works as expected and there are not side affects.
- Should now work in IE 9+
- Test down to Chrome 15.
- Ensure you can successfully run the services following the README?
- Ensure that the script in the browser is minified and if its JS, its ES5 syntax and not ES6 etc.

### Links
